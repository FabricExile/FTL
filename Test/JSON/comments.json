{"foo": //line comment
  true, /*block comment*/"bar":false}
