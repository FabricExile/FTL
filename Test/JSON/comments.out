1:1 OBJECT 2
  1:2 STRING 3 'foo'
    2:3 BOOLEAN true
  2:26 STRING 3 'bar'
    2:32 BOOLEAN false
